test/ros_parameter_handler_test.cpp

```diff
@@ -100,7 +104,7 @@ TEST_F(ROSParameterHandlerTest, test_required_params_only)
   ros::param::set("sensor_ip", sensor_ip_);
   ros::param::set("host_ip", host_ip_);
   ros::param::set("host_udp_port", host_udp_port_);
-
+  ros::param::set("skip", default_skip_);
```

Comment:
Why do we need to set the `skip`-parameter here when the test is called "required params only"?